Makefile

```diff
@@ -16,6 +16,7 @@ all: default install
 
 compile: ## Create the kubicorn executable in the ./bin directory.
 	go build -o bin/kubicorn -ldflags "-X github.com/kris-nova/kubicorn/cmd.GitSha=${GIT_SHA} -X github.com/kris-nova/kubicorn/cmd.Version=${VERSION}" main.go
+	docker build -t kris-nova/kubicorn:v0.0.003 . #docker image build.
```

Comment:
We should remove this from makefiles as it's assuming every user has Docker locally installed.